addons/note/static/src/scss/note.scss

```diff
@@ -48,6 +48,9 @@
         border: none;
     }
     .o_note_datetime {
+        .o_note_set_datetime {
```

Comment:
My bad, this element is not a child of `o_note_datetime`, so move it next to it:

```
.o_note_datetime {
   ...
}
.o_note_set_datetime {
   ...
}
```

Reply:
By the way I see the icon even with text muted. However it seems other people don't see it.